src/luxor_overrides.jl

```diff
@@ -273,3 +274,298 @@ function background(background_color)
     end
     Luxor.background(background_color)
 end
+
+function apply_transform(transform::Vector{Float64}, poly::Vector{Point})
+    retpoly = Point[]
+    for pt in poly
+        res = cairotojuliamatrix(transform) * [pt[1], pt[2], 1]
+        push!(retpoly, Point(res[1], res[2]))
+    end
+    retpoly
+end
+
+function Luxor.strokepath()
```

Comment:
Feels like the following four functions have a lot of the same function calls. Would be easier to read if another function is added which takes in inputs like `:stroke` or `:fill`

Reply:
yes a lot of repetitive code for the 4 functions , ill wrap them up to make it cleaner